rest_docs/rest-catalog-open-api.yaml

```diff
@@ -142,16 +142,7 @@ paths:
           example: "accounting%00tax"
       responses:
         200:
-          description: OK
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/responses/ListNamespacesResponse'
-              examples:
-                NonEmptyResponse:
-                  $ref: '#/components/examples/ListNamespacesNonEmptyExample'
-                EmptyResponse:
-                  $ref: '#/components/examples/ListNamespacesEmptyExample'
```

Comment:
These examples are defined on the response itself, so function the same and render the same.

That should be true of all in-line examples removed for any *Response.